composer.json

```diff
@@ -15,7 +15,8 @@
         "guzzlehttp/guzzle": "^6.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.1"
+        "phpunit/phpunit": "^6.1",
+        "illuminate/view": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*"
```

Comment:
I've included `illuminate/view` in `require-dev` to avoid downloading it on other frameworks and still allow this libs tests to run.